js_modules/dagit/packages/core/src/assets/__tests__/AssetPartitions.test.tsx

```diff
@@ -70,45 +70,43 @@ const SingleDimensionAssetPartitions: React.FC<{
 
 describe('AssetPartitions', () => {
   it('should support filtering a time-partitioned asset to a time range using the top bar', async () => {
-    await act(async () => {
-      render(<SingleDimensionAssetPartitions assetKey={{path: ['single_dimension_time']}} />);
-    });
+    render(<SingleDimensionAssetPartitions assetKey={{path: ['single_dimension_time']}} />);
 
     await waitFor(() => {
       expect(screen.getByTestId('partitions-selected')).toHaveTextContent('6,000 Partitions');
     });
 
-    const partitionInput = screen.getByTestId('dimension-range-input');
-    await userEvent.clear(partitionInput);
-    await userEvent.type(partitionInput, '{[}2022-11-28-20:00...2022-12-05-01:00{]}');
-    await userEvent.tab();
+    await act(async () => {
```

Comment:
do you need this act even though there's an waitFor below?

Reply:
Yes because it's important that we run these user events /in order/, and they are generating react re-renders as they run so they must be in an act block.